### PR TITLE
Add multiprocess training to image classification

### PR DESCRIPTION
**Pull request by LielinJiang**
Add multiprocess training to image classification.

mutlprocess train resnet18:
image
original resnet18 acc1:
image

mutlprocess train mobilenet_v2_1.0:
image
original mobilenet_v2 acc1：
image

### CODE REVIEW

File: PaddleCV/image_classification/scripts/train/ResNet50_dist.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+##Training details
+#GPU: NVIDIA® Tesla® V100 4cards 120epochs 67h
```
Comment (shippingwang) [inline PaddleCV/image_classification/scripts/train/ResNet50_dist.sh]: update the training info?

File: PaddleCV/image_classification/train.py [hunk 2]
```diff
@@ -167,25 +166,26 @@ def train(args):
 
     #init model by checkpoint or pretrianed model.
     init_model(exe, args, train_prog)
+    num_trainers = int(os.environ.get('PADDLE_TRAINERS_NUM', 1))
+    imagenet_reader = reader.ImageNetReader(0 if num_trainers > 1 else None)
+    train_reader = imagenet_reader.train(settings=args)
+    if args.use_mixup:
```
Comment (shippingwang) [inline PaddleCV/image_classification/train.py]: Can this "use mixup" branch move to ```image_reader.train```?

**Review by shippingwang — CHANGES_REQUESTED** [review]

**Comment by shippingwang** [comment]
and following #3480 and #3479

File: PaddleCV/image_classification/scripts/train/ResNet50_dist.sh (hunk 1 above)
Comment (LielinJiang) [reply PaddleCV/image_classification/scripts/train/ResNet50_dist.sh]: have not tested this completely, just removed.

File: PaddleCV/image_classification/train.py (hunk 2 above)
Comment (LielinJiang) [reply PaddleCV/image_classification/train.py]: yes, done.

**Comment by LielinJiang** [comment]
> and following #3480 and #3479

done

**Review by shippingwang — APPROVED** [review]
good job!

**Pull request merged**